reproducible/ubuntu/overlay/etc/machine-id

```diff
@@ -0,0 +1 @@
+793274e99ef692a48f77ea0eea7bd0cb
```

Comment:
This appears to be empty upstream, could you keep this file but make it empty as well?

Reply:
done